cs251tk/student/markdownify/check_submit_date.py

```diff
@@ -16,13 +16,13 @@ def check_dates(spec_id, username, spec, basedir):
         for file in spec['files']:
 
             # Run a git log on each file with earliest commits listed first
-            _, res, _ = run(['git', 'log', '--reverse',
+            _, res, _ = run(['git', 'log', '--reverse', '--pretty=format:%ad', '--date=iso8601',
                              os.path.join(basedir, file['filename'])])
 
             # If we didn't get an error, add date to array
             if res[0] is not 'f':

```

Comment:
The first value returned is the success/failure of the command. If it is successful, it returns "success", otherwise it returns a string for the error as defined in the run_static() function.